files/fr/mdn/community/contributing/translated_content/index.md

```diff
@@ -38,7 +38,7 @@ Nous avons actuellement dégelé les langues suivantes :
 ### Russe (ru)
 
 - Discussions : [Matrix (#mdn-l10n-ru channel)](https://chat.mozilla.org/#/room/#mdn-l10n-ru:mozilla.org)
-- Contributeurs actuels : [armanpwnz](https://github.com/armanpwnz), [captainspring](https://github.com/captainspring), [mpstv](https://github.com/mpstv), [myshov](https://github.com/myshov), [Saionaro](https://github.com/Saionaro), [sashasushko](https://github.com/sashasushko), [lex111](https://github.com/lex111)
+- Contributeurs actuels : [captainspring](https://github.com/captainspring), [mpstv](https://github.com/mpstv), [myshov](https://github.com/myshov), [Saionaro](https://github.com/Saionaro), [sashasushko](https://github.com/sashasushko), [lex111](https://github.com/lex111)
```

Comment:
It doesn't seem like `mpstv` should be in the list any longer:

```suggestion
- Contributeurs actuels : [captainspring](https://github.com/captainspring), [myshov](https://github.com/myshov), [Saionaro](https://github.com/Saionaro), [sashasushko](https://github.com/sashasushko), [lex111](https://github.com/lex111)
```

Reply:
It seems that the GitHub account [`captainspring`](https://github.com/captainspring) is deleted, hi @sashasushko should we also remove this?

Reply:
Looks like that account has changed their username, see #13014!

Reply:
Yeah, let's also sync this :)